lib/server.go

```diff
@@ -373,6 +373,10 @@ func (s *Server) initDefaultCA(renew bool) error {
 	if err != nil {
 		return err
 	}
+	if ca.issuer.NonceManager() != nil {
```

Comment:
We don't want to start the nonce sweeper when we are just initializing the CA. This should be called only when we start the server, maybe the `Start` method might be the appropriate place for this. 

Also, this applies to more than just the default CA. Take a look at the method `initMultiCAConfig`, a single server can contain multiple CA so this needs to apply to all of them.